### PR TITLE
Create proposed-decisions.md

### PR DESCRIPTION
**Pull request by MarcelUntied**
This is a document/page with (proposed) decisions. It summarizes the outcome of the discussed issues

### CODE REVIEW

File: proposed-decisions.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- Folio (incl. paper vouchers)
+    - Partners (SA, MP) within the eco-system have to evaluate the impact of the proposed solutions on their supply chain (forecasts/prognosis, purchase, stock, etc)
+- ECK
+
+[Related Issue 18](https://github.com/stichtingsem/functional-overview/issues/18)
+
+### Out of scope (for now):
+- Setup & Support
+    - Most important next step in the SEM pilot
+    - More decentralized = more risk > How to reduce risk?
```
Comment (eliashassing154) [inline proposed-decisions.md]: I don't agree with the notion of decentralization equals more risk, quite the reverse (it's also out of scope, but not sure what risk you're referring to).

File: proposed-decisions.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+- Communication 
+- School processes / administration (SIS, ELO, IDP)
+    - Not enrolled students (0e Jaars)
+- Teacher licenses (including trial licenses)
+    - Additional services for teachers > prepare access locations for teacher licenses
+- Kennisnet ARP administration
+- Progress & Results (Part of Use Case 4.0)
```
Comment (eliashassing154) [inline proposed-decisions.md:114]: I would include this (though possibly later) in scope, as discussed today.

File: proposed-decisions.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+- NON ECK
+
+## What is Fulfilment Update LMS
+[Related Issue 19](https://github.com/stichtingsem/functional-overview/issues/19)
+
+Update contains contains information about order confirmations, forecasts and availability
+
+## Usage Information blended learning
+[Related Issue](https://github.com/stichtingsem/functional-overview/issues/20)
+
```
Comment (eliashassing154) [inline proposed-decisions.md:128]: To clarify: it wouldn't matter if it's a blended or pure digital product

File: proposed-decisions.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+# (PROPOSED) DECISIONS
+
+## Service credits still necessary in ecosystem?
+[Related Issue 9](https://github.com/stichtingsem/functional-overview/issues/9)
+
+Service credits will **not** be necessary in the new ecosystem. The functionality of service credits will be replaced by entitlements. If there is **no** entitlement or the quantity of the entitlement is **not** sufficient there will be **no** access for the user.
+
```
Comment (eliashassing154) [inline proposed-decisions.md:7]: In the context of a user, it's important to note that when there aren't enough entitlements left, an administrator can resolve that within 10 minutes through the proper channels: order additional entitlements on MP, .. etc

**Review by eliashassing154 — CHANGES_REQUESTED** [review]

File: proposed-decisions.md (hunk 1 above)
Comment (MarcelUntied) [reply proposed-decisions.md]:
```suggestion
    - More decentralization asks for higher maturity level regarding to maintenance and fixation of stem data and identity 
      - How to guard data quality?
```

File: proposed-decisions.md (hunk 2 above)
Comment (MarcelUntied) [reply proposed-decisions.md:114]: We discussed this within the stream and in our opinion this should be out of scope, Let's discuss this tomorrow in the steering committee

File: proposed-decisions.md (hunk 3 above)
Comment (MarcelUntied) [reply proposed-decisions.md:128]: I related this to the original issue, but usage information will always (full digital and blended) be communicated in the ecosystem

File: proposed-decisions.md (hunk 4 above)
Comment (MarcelUntied) [reply proposed-decisions.md:7]: You are right, but it is good to make a clear statement that there has to be an entitlement to get access. It is the responsibility of the school and market place to arrange this in time

**Comment by eliashassing154** [comment]
@MarcelUntied (data) quality assurance might be something we want to address as a separate topic as I agree that it will be crucial for a successful pilot/implementation. We can define a scenario to test the setup process to address this? Input controls and setup validation will be relevant controls I imagine.

**Comment by MarcelUntied** [comment]
I think validation of the setup process is a good way to ensure (at front) that everything is ready for use.

**Comment by eliashassing154** [comment]
Agreed, but we're currently not testing that in the functional stream so let's discuss in tomorrow's steering committee.

**Review by cliftonc — COMMENTED** [review]
Looks good as a summary.

**Comment by MarcelUntied** [comment]
@eliashassing154 @cliftonc Can we commit/end this pull request zo the document is visible for the community?

**Comment by cliftonc** [comment]
I've approved.

**Review by eliashassing154 — APPROVED** [review]

**Review by eliashassing154 — APPROVED** [review]

**Pull request merged**